.github/workflows/test-setup-sigstore-env.yml

```diff
@@ -0,0 +1,47 @@
+name: test setup-sigstore-env
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'actions/setup-sigstore-env/**'
+  pull_request:
+    paths:
+      - 'actions/setup-sigstore-env/**'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+      - id: setup-sigstore-env
+        uses: ./actions/setup-sigstore-env
+      # Pending https://github.com/sigstore/cosign-installer/issues/187.
```

Comment:
I cut a new release https://github.com/sigstore/cosign-installer/releases/tag/v3.8.2

It would be nice if we could reuse the Cosign container downloaded in the script rather than download the binary, we can follow up on this later.